packages/camera/camera/README.md

```diff
@@ -19,6 +19,8 @@ First, add `camera` as a [dependency in your pubspec.yaml file](https://flutter.
 
 ### iOS
 
+iOS 10.0 of higher is needed to use the camera plugin. If compiling for any version lower than 10.0 make sure to check the iOS version with for example the [device_info plugin](https://pub.dev/packages/device_info)
```

Comment:
nit: I think the explanation can be improved a little bit by converting the sentence to:

```suggestion
iOS 10.0 of higher is needed to use the camera plugin. If compiling for any version lower than 10.0 make sure to check the iOS version before using the camera plugin. For example, using the [device_info](https://pub.dev/packages/device_info) plugin.
```